dotcom-rendering/src/web/components/ContainerLayout.tsx

```diff
@@ -34,6 +34,7 @@ type Props = {
 	format?: ArticleFormat;
 	ophanComponentName?: string;
 	ophanComponentLink?: string;
+	containerName?: string;
```

Comment:
I'm not super strong on this so feel free to ignore me but is adding this prop increasing the cognitive complexity of things here? This component has a lot of props!

Maybe if it were marked as debug only somehow then future developers would not need to worry about what it is?

Reply:
Yeah I didn't love this either - the other name I'd considered was `dataContainerName` to try and suggest its just there for a data attribute, but we could also try `debugContainerName` - my fear here was introducing a new pattern for debug properties could increase the cognitive load just as much if you're a developer trying to figure out what that means.

Reply:
Would it work if we used the `ophanComponentName` for debugging?

Reply:
Fair